content/en/docs/concepts/workloads/pods/downward-api.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Downward API
+content_type: concept
+description: >
+  There are two ways to expose Pod and container fields to a running container:
```

Comment:
In the API, `Container` with a capital C doesn't appear. We do see that in the Golang implementation, but not in the API (where it's actually `containers`).

Reply:
They are API definitions, though not first class API resources.

https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.23/#container-v1-core
https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#Container

Reply:
I'm going to look for a way to word this so that people don't start trying to `kubectl get container` and expect it to work, but also that readers end up clear on the two kinds of thing they can reference using the downward API.